.github/workflows/appsec.yml

```diff
@@ -250,3 +250,17 @@ jobs:
       - run: yarn test:integration:appsec
       - uses: ./.github/actions/node/latest
       - run: yarn test:integration:appsec
+
+  passport:
```

Comment:
does that mean our passport tests were not running before ? or you just created the first ones ?

Reply:
These are the first ones.